src/obr/signac_wrapper/operations.py

```diff
@@ -21,11 +22,36 @@
 
 
 class OpenFOAMProject(flow.FlowProject):
+    filtered_jobs = None
+    queried_jobs = None
+
     def print_operations(self):
         ops = sorted(self.operations.keys())
         logging.info("Available operations are:\n\t" + "\n\t".join(ops))
         return
 
+    def get_jobs(self, filter=Union[None, dict[str, str]], queries=None):
+        self.filtered_jobs = self.filter_jobs(filters=filter)
+        resulting_jobs = self.query_jobs()
+        return resulting_jobs
+
+    def filter_jobs(self, filters: dict[str, str], output=True):
+        all_jobs: list[Job] = self.queried_jobs or self.find_jobs()
+        # chain filtering of jobs
+        for key, value in filters.items():
+            all_jobs = filter(
+                lambda job: value in self.get_job_status(job)[key], all_jobs
```

Comment:
I have trouble understanding this, but I guess here `core.queries.filter_jobs_by_query` will be used?